events/YYMMDD-example-event.md

```diff
@@ -0,0 +1,35 @@
+# Introduction
+Awesome that you are interested in hosting an event!
+
+This is a template that we use here at Stockholm React JS.
+Since we parse this file programatically, please **do not** change headers or bold text, but feel free to change any text inside of a section or add anything you find appropriate.
+
+# Your event intro
+{{The intro text of this event, use this to quickly explain what the event
+will be all about, be clear and try to get peoples attention. Don't write too much.}}
+
+# Hosted by
+**Company Name:** {{Name of the company host}}
+
+**Address:** {{Host Address}}
+
+# Full length talks
+**By:** {{Name of speaker}}
+
+**Title:** {{Title of the talk}}
+
+**Link:** {{Any link}}
+
+# Lightning talks
+**By:** {{Name of speaker}}
+
+**Title:** {{Title of the lightning talk}}
+
+# Food / Drinks
+**Food:** {{Comma seperated list of different food}}
```

Comment:
Food and drinks might not be available at the time we want to post the event, should we mark these as optional somehow? Either here or in the description?